pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -536,7 +536,7 @@ func (c *Compactor) runCompact(ctx context.Context, logger log.Logger, job Job,
 		}
 
 		fpRate := c.limits.BloomFalsePositiveRate(job.tenantID)
-		resultingBlock, err = compactNewChunks(ctx, logger, job, fpRate, bt, storeClient.chunk, builder)
+		resultingBlock, err = compactNewChunks(ctx, logger, job, fpRate, bt, storeClient.chunk, builder, c.limits)
```

Comment:
I believe it's better to pass the limits because we will always add additional parameters and it will make us to add additional function's incoming parameters... I will remove `fpRate` and will resolve it inside the function.